commerce-account-web/src/main/java/com/liferay/commerce/account/web/internal/portlet/action/EditCommerceAccountUserMVCActionCommand.java

```diff
@@ -218,92 +229,157 @@ protected void updatePassword(
 			ActionResponse actionResponse)
 		throws Exception {
```

Comment:
wow... this seems like a lot of logic for us to maintain.... this is the only way to expose this functionality?

Reply:
I agree, what do you suggest to simplify?